rmk-macro/src/input_device/adc.rs

```diff
@@ -73,14 +73,24 @@ pub(crate) fn expand_adc_device(
                 });
                 let joy_ident = format_ident!("joystick_processor_{}", joystick.name);
                 processor_name.push(quote!(#joy_ident));
+
+                let map_to = match joystick.map_to.unwrap_or("mouse".into()).as_str() {
+                    "mouse" => quote! { rmk::input_device::joystick::MapTo::Mouse },
+                    "joystick" => quote! { rmk::input_device::joystick::MapTo::Joystick },
+                    unsupported => {
+                        panic!("Unsupported map_to type: {}", unsupported)
```

Comment:
Is there better, more elegant way to handle compile error here?

Reply:
panic at the compile-time is fine. Adding something making users to know which configuration is wrong would be better